### DISCUSSION TITLE
I tried to use this template workspace (Cuda/PyTorch/TensorFlow) and got no space left in device through installation

### DISCUSSION THREAD

**Discussion by alsmadi** [coder/coder#14791] [General]
Version: [`v2.14.2+d1cd784`](https://github.com/coder/coder/commit/d1cd7848665516617f0f24ecc0b0607c6bdd8408)

I tried to use this template workspace (Cuda/PyTorch/TensorFlow) and got no space left in device through installation

**Comment by stirby** [comment]
For resource requirements, I'd start with our [architecture docs](https://coder.com/docs/architecture/architecture). Were you using one of our example templates?
